tests/network/libs/ip.py

```diff
@@ -0,0 +1,49 @@
+import random
+from functools import cache
+
+MAX_NUM_OF_SECONDARY_ETH_INTERFACES = 4
```

Comment:
Please rename in the context of random subnets, not in the context of what it is used for.
This module does not know who uses it.